src/etl/NFTHelpers.cpp

```diff
@@ -356,4 +356,22 @@ getNFTDataFromObj(std::uint32_t const seq, std::string const& key, std::string c
 
     return nfts;
 }
+
+std::vector<NFTsData>
+getUniqueNFTsDatas(std::vector<NFTsData> const& nfts)
+{
+    std::vector<NFTsData> results = nfts;
+    // Remove all but the last NFTsData for each id. unique removes all but the first of a group, so we want to
```

Comment:
This could be a doxygen note instead perhaps? I mean, the code speaks for itself tbh

Reply:
At the very least, it needs a line of space before the comment here or move the comment all the way up